docs/license.rst

```diff
@@ -0,0 +1,132 @@
+.. _license-page:
+
+License Information
+===================
+
+Source Code
+^^^^^^^^^^^
+
+All model code is held under the `BSD-3 Clause License <https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE>`_.
```

Comment:
```suggestion
All model source code is held under the `BSD-3 Clause License <https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE>`_.
```

Reply:
Not using `license.rst` anymore, but I have put this snippet at the top of `pretrained.rst` with the suggested change.